types/int.go

```diff
@@ -14,6 +14,28 @@ type Int struct {
 	sql.NullInt64
 }
 
+// TransformIntZeroToNull transforms a valid Int carrying a zero value to a SQL NULL.
+func TransformIntZeroToNull(i *Int) {
```

Comment:
I would rename this to `TransformZeroIntToNull` as it is also `EmptyString` and not `StringEmpty`. I could also imagine that we could introduce a generic version `TransformZeroValueToNull` later.

Reply:
Renamed.